diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 9d09992..f5348e3 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -36,4 +36,5 @@ target_link_libraries(rnscreens
     ReactAndroid::jsi
     fbjni::fbjni
     android
+    c++_shared
 )
diff --git a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
index b2dfc56..63bd08a 100644
--- a/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt
@@ -44,6 +44,7 @@ target_link_libraries(
   ReactAndroid::reactnative
   ReactAndroid::jsi
   fbjni::fbjni
+  c++_shared
 )
 
 if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 80)
